resources/multisite_drupal_standard.make

```diff
@@ -364,7 +364,7 @@ projects[mailsystem][subdir] = "contrib"
 projects[mailsystem][version] = "2.34"
 
 projects[maxlength][subdir] = "contrib"
-projects[maxlength][version] = "3.2"
+projects[maxlength][version] = "3.2-beta2"
```

Comment:
Is this actually a downgrade of a module (from stable release to a 2nd beta release) or do they just follow a weird release scheme?